util/collections/machine_collection_test.go

```diff
@@ -166,10 +166,10 @@ func machine(name string, opts ...machineOpt) *clusterv1.Machine {
 
 func machines() collections.Machines {
 	return collections.Machines{
-		"machine-4": machine("machine-4", withCreationTimestamp(metav1.Time{Time: time.Date(2018, 04, 02, 03, 04, 05, 06, time.UTC)})),
-		"machine-5": machine("machine-5", withCreationTimestamp(metav1.Time{Time: time.Date(2018, 05, 02, 03, 04, 05, 06, time.UTC)})),
-		"machine-2": machine("machine-2", withCreationTimestamp(metav1.Time{Time: time.Date(2018, 02, 02, 03, 04, 05, 06, time.UTC)})),
-		"machine-1": machine("machine-1", withCreationTimestamp(metav1.Time{Time: time.Date(2018, 01, 02, 03, 04, 05, 06, time.UTC)})),
-		"machine-3": machine("machine-3", withCreationTimestamp(metav1.Time{Time: time.Date(2018, 03, 02, 03, 04, 05, 06, time.UTC)})),
+		"machine-4": machine("machine-4", withCreationTimestamp(metav1.Time{Time: time.Date(2018, 0o4, 0o2, 0o3, 0o4, 0o5, 0o6, time.UTC)})),
```

Comment:
This seems weird, those are not octal number

Reply:
I think based on the Golang specification for integer literals, any integer with a leading zero is treated as octal. But since Time.Date takes in Int, these are implicitly converted to their corresponding integer value. And also since they are all below 8, oct(n) = dec(n). I vote to removing the leading zeros in the `Date` method to avoid this.

Reply:
Done